setup.py

```diff
@@ -61,6 +64,12 @@
     )
     ext_modules.append(ext)
 
+# Hack to install ta-lib library loading into virtualenv
+def install_virtualenv_lib_loader():
+    with open(os.getenv('VIRTUAL_ENV', './venv') + '/bin/activate', 'a') as f:
```

Comment:
I found a page that talks about using ``LD_RUN_PATH`` for this:

https://groups.google.com/forum/#!topic/python-virtualenv/-U4epR1YWFk

Also, not something I expect you to have to work on, but would be nice if it worked on Mac, Linux, and Windows like the ``talib`` library does.